lib/resources/dstu2/condition.yaml

```diff
@@ -101,10 +101,13 @@ fields:
     terminology:
     - display: SNOMED CT
       system: http://snomed.info/sct
```

Comment:
fixing per Bradley's comments above. This will likely change once all resources use the new format.